README.md

```diff
@@ -38,3 +38,18 @@ If you are making a major refactor, changing interfaces, or any other breaking c
 Once approved, we will merge your PR and publish a new version. You can then update the version number in your `composer.json` file and deploy your changes.
 
 All breaking changes will be published as a major version. If you are using a major version, you will need to update your code to use the new interfaces. We don't expect that to occur often.
+
+## Creating a new release
+
+**Only applies to Automatticians**
```

Comment:
is there a special user group who has access to do this, or is it anyone/everyone in the Automattic team here on github?

Reply:
Invited external users will have that access IIRC. I'd remove this line altogether because total strangers won't be able to make a release in the first place.

Reply:
> I'd remove this line altogether because total strangers won't be able to make a release in the first place.

Isn't that the case that it only applies to a12s then? The invited external users are the exception, and I have never seen we do that.

Reply:
I could change it to "This only applies to folks with write permissions" but I think this would be more confusing for both external users and a12s where they will think "do I have these permissions?". So unless there is a better suggestion, I rather keep it like this.